docker/sandbox/build.sh

```diff
@@ -65,7 +65,11 @@ generate_tag() {
 
 detect_repo() {
     OS_SEP_BY_SLASH=${OS/-//}
-    REPO=${REPO:-"http://bigtop-repos.s3.amazonaws.com/releases/${BIGTOP_VERSION}/${OS_SEP_BY_SLASH}/x86_64"}
+    if [ $HOSTTYPE = "aarch64" ] ; then
+       REPO=${REPO:-"http://bigtop-repos.s3.amazonaws.com/releases/${BIGTOP_VERSION}/${OS_SEP_BY_SLASH}/aarch64"}
+    else
+       REPO=${REPO:-"http://bigtop-repos.s3.amazonaws.com/releases/${BIGTOP_VERSION}/${OS_SEP_BY_SLASH}/x86_64"}
+    fi
```

Comment:
It seems we can put HOSTTYPE in the REPO variable directly since I got exactly "x86_64" on x86_64 systems.

REPO=${REPO:-"http://bigtop-repos.s3.amazonaws.com/releases/${BIGTOP_VERSION}/${OS_SEP_BY_SLASH}/$HOSTTYPE"}

How do you think?